recipes-extended/images/pelux-image-update/raspberrypi3/sw-description

```diff
@@ -5,7 +5,7 @@ software = {
     stable : {
         main : {
             images: ({
-                filename = "core-image-pelux-minimal-raspberrypi3.ext3.gz";
+                filename = "PELUX_IMAGE";
```

Comment:
What is the motivation behind this name change? I guess it's the python script?

Reply:
Basically, this sw-description used to be specific to "core-image-pelux-minimal" target so it hardcoded the name of the generated rootfs: core-image-pelux-minimal-raspberrypi3.ext3.gz
However, since we would like to have one sw-description for every target, we can't hardcode the name of one specific .ext3.gz
This "PELUX_IMAGE" placeholder, both documented in the .inc file and in the commit message, is dynamically replaced by my python script with "core-image-pelux-minimal-raspberrypi3.ext3.gz" or "core-image-pelux-minimal-dev-raspberrypi3.ext3.gz" or "core-image-pelux-qtauto-neptune-raspberrypi3.ext3.gz" or "core-image-pelux-qtauto-neptune-dev-raspberrypi3.ext3.gz" or whatever we could need next...

Reply:
Yup got the point thanks!

Reply:
I'm not saying this is a beautiful solution I am proud of but I couldn't find anything better given the constraints of meta-swupdate :smiley: